php/WP_CLI/CommandWithTranslation.php

```diff
@@ -125,6 +143,106 @@ public function install( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Updates the active translation of core, plugins, and themes.
+	 *
+	 * [--dry-run]
+	 * : Preview which translations would be updated.
+	 *
+	 * @subcommand update
+	 */
+	public function update( $args, $assoc_args ) {
+
+		// Ignore updates for the default locale.
+		if ( 'en_US' == get_locale() ) {
+			\WP_CLI::success( "Translations updates are not needed for the 'English (US)' locale." );
```

Comment:
I thought as the command deals with translations of core, plugins and themes, it would be "Translations" (plural).